lib/puma/minissl.rb

```diff
@@ -308,5 +307,5 @@ def close
         @socket.close unless @socket.closed?       # closed? call is for Windows
       end
     end
-  end
+  end if IS_JRUBY || (Puma::MiniSSL.check.nil? rescue nil)
```

Comment:
This should be a better check, silently skipping tests if e.g. there is a typo, seems bad.

Reply:
That check is to not load minissl.rb if MRI Puma wasn't compiled with SSL support.  Similar checks for some of the tests also.  The silent skips will only happen when Puma isn't compiled with SSL support.  I'm working with it in my fork (adding a 'none SSL' job) and locally.  JFYI, working with JRuby head locally.

Reply:
I think we can leave a comment (or add a new method to Puma detect) to explain.

Reply:
Ok.  Let me think about that (MRI/JRuby/TruffleRuby).